pkg/apis/wksprovider/machine/os/os.go

```diff
@@ -17,6 +17,7 @@ import (
 	"time"
 
 	ssv1alpha1 "github.com/bitnami-labs/sealed-secrets/pkg/apis/sealed-secrets/v1alpha1"
+	"github.com/ghodss/yaml"
```

Comment:
this version is over a year old and sigs.k8s.io/yaml replaced it.  is there a specific reason we need the older version?

Reply:
Thanks, I was missing the context, it was copied over from other parts of the code. Changed to `sigs.k8s.io/yaml`